File: src/main/java/shirokuro/embedscript/Prefix.java
package shirokuro.embedscript;

import org.bukkit.ChatColor;

/**
 * @author shirokuro
 */
public final class Prefix {
    public static final String PREFIX = ChatColor.GRAY + "<"
        + ChatColor.DARK_GREEN + "E" + ChatColor.GREEN + "S" + ChatColor.GRAY + "> ";
    public static final String WARN_PREFIX = PREFIX + ChatColor.GOLD;
    public static final String ERROR_PREFIX = PREFIX + ChatColor.RED;
    public static final String SUCCESS_PREFIX = PREFIX + ChatColor.GREEN;

    private Prefix() {
        throw new UnsupportedOperationException("Constant class.");
    }
}

File: src/main/kotlin/com/github/kuro46/embedscript/Prefix.kt
package com.github.kuro46.embedscript

import org.bukkit.ChatColor

/**
 * @author shirokuro
 */
object Prefix {
    val PREFIX = (ChatColor.GRAY.toString() + "<"
            + ChatColor.DARK_GREEN + "E" + ChatColor.GREEN + "S" + ChatColor.GRAY + "> ")
    val WARN_PREFIX = PREFIX + ChatColor.GOLD
    val ERROR_PREFIX = PREFIX + ChatColor.RED
    val SUCCESS_PREFIX = PREFIX + ChatColor.GREEN
}
